app/src/main/res/values/privacy_policy.xml

```diff
@@ -0,0 +1,4 @@
equest as soon asreasonably practicable. Please note that if you opt out, we may still send you important administrative messages, from which you cannot opt out.</p><p>If you are a resident of California, under 18, and a registered user of the Site, you may ask us to remove content or information that you have posted to the Site by writing to admin@oppia.org. Please note that your request doesnot ensure complete or comprehensive removal of the content or information, as, for example, some of your content may have been reposted by another user.</p><h2>Use by children</h2><p>The Site is not directed to individuals under the age of thirteen (13). If you are under 13, we do not want your Personal Information and you should not provide it to us. If you are a legal guardian and believe that your childwho is under 13 has provided us with Personal Information, please contact us at admin@oppia.org to have your child\'s information removed.</p><h2>Jurisdictional issues</h2><p>The Site is controlled and operated by us from the United States, and is not intended to subject us to the laws or jurisdiction of any state, country or territory other than that of the United States. Your Personal Informationmay be stored and processed in any country where we or our service providers have facilities, and by using the Site you consent to the transfer of information to countries outside of your country of residence, including theUnited States, which may have different data protection rules than those of your country. In certain circumstances, courts, law enforcement agencies, regulatory agencies or security authorities in those other countries may beentitled to access your Personal Information.</p><h2>Links to third-party sites</h2><p>Our Site and content may feature links to third party websites that offer goods, services or information. When you click on one of these links, you will be leaving our Site and will no longer be subject to this Privacy Policy.We are not responsible for the information or other practices of the other sites that you visit and urge you to review their privacy policies before you provide them with any personally identifiable information. Third partysites or services may collect and use information about you in a way that is different from this Privacy Policy.</p><h2>Changes and updates to this Privacy Policy</h2><p>Oppia will continue to evolve and we may update our Privacy Policy from time to time to reflect our practices. The "Last updated" legend at the bottom of this Privacy Policy indicates when this Privacy Policy was last revised,and any changes will become effective immediately upon posting of the Privacy Policy on the Site. Your continued use of the Site following these changes constitutes your agreement to the revised Privacy Policy.</p><h2>Contacting Us</h2><p>If you have questions about this Privacy Policy or would like to provide feedback, please send an e-mail to admin@oppia.org.</p><em>Last updated: 24 May 2018</em>]]></string>
```

Comment:
There seem to be a bunch of spacing issues here. For example, ``revised,and`` should be ``revised, and``, ``Privacy Policy.We are`` should be ``Privacy Policy. We are``, ``Informationmay`` should be ``Information may``, etc. Please update the policy to match the policy including spacing between words.

Reply:
This was taken from web. I need to correct everywhere wherever there is space.

Reply:
Did you copy this from the actual website or the raw pages? The latter (i.e. https://github.com/oppia/oppia/blob/develop/core/templates/pages/privacy-page/privacy-page.component.html & https://github.com/oppia/oppia/blob/develop/core/templates/pages/terms-page/terms-page.component.html) would probably be easier for correctness, and are also the pages the later script need to verify against.

Reply:
I copied from these above link and updated the page.